integration-cli/docker_cli_run_test.go

```diff
@@ -3297,7 +3297,7 @@ func (s *DockerSuite) TestRunContainerNetModeWithExposePort(c *check.C) {
 	}
 
 	out, _, err = dockerCmdWithError("run", "--expose", "5000", "--net=container:parent", "busybox")
-	if err == nil || !strings.Contains(out, "Conflicting options: --expose and the network mode (--expose)") {
+	if err == nil || !strings.Contains(out, "Conflicting options: --expose and the network mode (--net)") {
```

Comment:
@duglin I think the original code meant --net instead. I corrected accordingly. It should not be --expose. Nothing to do with error codes